examples/Transfer_Learning/basic_transfer_learning.py

```diff
@@ -0,0 +1,175 @@
+## Transfer Learning
+
+# This example demonstrates BayBE's
+# {doc}`Transfer Learning </userguide/transfer_learning>` capabilities:
+# * We construct a campaign,
+# * give it access to data from a related but different task,
+# * and show how this additional information boosts optimization performance.
+
+### Imports
+
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from botorch.test_functions.synthetic import Hartmann
+
+from baybe import Campaign
+from baybe.objective import Objective
+from baybe.parameters import NumericalDiscreteParameter, TaskParameter
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget
+from baybe.utils.botorch_wrapper import botorch_function_wrapper
+from baybe.utils.plotting import create_example_plots
+
+### Settings
+
+# The following settings are used to set up the problem:
+
+SMOKE_TEST = "SMOKE_TEST" in os.environ
```

Comment:
its weird that all other constants have an explanation and this not, indeed a user might wonder what this is and why is it here, I suggest `# used to reduce the problem complexity in continuous integration pipelines`

Reply:
I can already hear **somebody** arguing that people will know what a SMOKE_TEST is :P But since this someone is not here, and since I agree with you, I'll change ;)